src/plugins/discover/public/application/helpers/get_sharing_data.ts

```diff
@@ -19,7 +19,10 @@ const getSharingDataFields = async (
   timeFieldName: string,
   hideTimeColumn: boolean
 ) => {
-  if (selectedFields.length === 1 && selectedFields[0] === '_source') {
+  if (
+    selectedFields.length === 0 ||
```

Comment:
This fixes the bug that was immediately noticed: only the date field is populated when exporting a new search (if no other fields are selected as columns)